src/main/java/core/basesyntax/fruitshop/file/reader/TaskReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.fruitshop.file.reader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskReaderImpl implements TaskReader {
+
+    @Override
+    public List<String> readFile(String fromFile) {
+        List<String> tempStorage = new ArrayList<>();
+
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(fromFile))) {
+            reader.readLine();
```

Comment:
our reader should not know from which file we are reading - csv or non-csv
so let's remove skipping first line from here and skip it later when parsing
```suggestion
```